tinyblog/articles/2016-07-01-registration.md

```diff
@@ -0,0 +1,72 @@
+# 2016 Registration
+
+> Published on 1st Jul 2016 by Ola Sitarska
+
+Every year tickets to Django: Under The Hood are selling out in a matter of minutes.
+This year we're trying something different to make the distribution of tickets
+a little bit more fair and less frustrating for everyone.
+
+Read on to learn about the important changes we're making this year!
+
+---
+
+#### The Short Version
+
+1. __On Tuesday, July 12, we're going to open registration.__
+You'll have two weeks to submit your e-mail address for the lottery, until July 19th at noon CET.
+
+2. __On Thursday July 21st at noon CET, we're going to randomly select attendees 
+who will receive a link to purchase their conference tickets.__ This link will be
+valid for 72 hours, and then we will randomly assign it to another attendee until
+all of the tickets have been purchased.
+
+That's it! ✨
+
+#### Pricing
+
+For 2016, the full conference passes including 1.5 day of Talks and 1.5 of 
+Sprints will have the following pricing:
+
+- 200€ for individual attendees (paying out of their own pocket) 
+- 300€ for corporate tickets (companies buying tickets for their employees)
+
+We're following DjangoCon's pricing structure to keep event as affordable 
+for everyone as possible.
+
+#### Reserve tickets for your team
+
+If you want to make sure that tickets for your team are reserved and set aside,
+we'd be happy to discuss a sponsorship opportunity with you. Please get in touch
+with us on [hello@djangounderthehood.com](mailto:hello@djangounderthehood.com).
+
+#### The Slightly Longer Version
+
+Django: Under The Hood started as this little thing for around 100 people who joined
+us in Amsterdam for the first time about 2.5 years ago. We had no idea if this 
+would work, but your excitement and care never ceased to surprise us.
+
+The 100 tickets for our very first edition were gone in just few minutes. On our second year,
+we welcomed 3 times as many people, yet the "sold out" banner showed up on our
+tickets page just 10 minutes after registration was open.
+
+The 300 is a number that we're capping the conference at. A sweet spot for serendipity, 
```

Comment:
We are capping the conference at 300 attendees. It's a good balance between having as many attendees as possible and making it possible to organise with our small volunteer team.